propolis/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-libc = "0.2"
+libc = "0.2.103"
```

Comment:
Curious, was there something specific in .103 that was added?

Reply:
We should have specified the patch version when the preadv/pwritev stuff was added.  Otherwise local cargo lock files would potentially cause build issues (at least while we're not checking in Cargo.lock)

Reply:
It wouldn't compile without 0.2.102 explicitly set (0.2 didn't work), and I bumped it when I saw the latest